odl/contrib/fom/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+__all__ = ()
+
+from .supervised_fom import *
```

Comment:
This also means that supervised_fom.py should be renamed to supervised.py, right?